examples/simplicial_2d_dg/elixir_euler_sbp_triangular_mesh_flux_diff.jl

```diff
@@ -0,0 +1,53 @@
+# !!! warning "Experimental features"
+
+using StartUpDG
+using Trixi, OrdinaryDiffEq
+
+polydeg = 3
+rd = RefElemData(Tri(), SBP(), polydeg)
+
+volume_flux = flux_ranocha
+dg = DG(rd, nothing #= mortar =#,
+        SurfaceIntegralWeakForm(FluxLaxFriedrichs()), VolumeIntegralFluxDifferencing(volume_flux))
+
+equations = CompressibleEulerEquations2D(1.4)
+initial_condition = initial_condition_convergence_test
+source_terms = source_terms_convergence_test
+
+# example where we tag two separate boundary segments of the mesh
+top_boundary(x,y,tol=50*eps()) = abs(y-1)<tol
+rest_of_boundary(x,y,tol=50*eps()) = !top_boundary(x,y,tol)
+is_on_boundary = Dict(:top => top_boundary, :rest => rest_of_boundary)
+VX, VY, EToV = StartUpDG.uniform_mesh(Tri(), 4)
+mesh = VertexMappedMesh(VX, VY, EToV, rd, is_on_boundary = is_on_boundary)
+
+boundary_condition_convergence_test = BoundaryConditionDirichlet(initial_condition)
+boundary_conditions = (; :top => boundary_condition_convergence_test,
+                        :rest => boundary_condition_convergence_test)
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, dg,
+                                    source_terms = source_terms,
+                                    boundary_conditions = boundary_conditions)
+
+tspan = (0.0, .1)
```

Comment:
```suggestion
tspan = (0.0, 0.1)
```

Reply:
How many time steps do you use for this?

Reply:
It took 6 timesteps.

Reply:
Hmm, might you try a few more time steps for CI if it doesn't take too long?

Reply:
Can do. I'll extend to final time `0.4`.